tests/__fixtures__/test.mocha.js

```diff
@@ -1,22 +1,44 @@
 const [, , , chaiToMatchScreenshot, browserWSEndpoint] = process.argv;
 const {Assertion, expect} = require('chai');
+const {withEyes} = require('../../mocha');
```

Comment:
We're running this test in a different process and passing other links to require via arguments. Not sure this is the best solution but we might want to be aligned with the same method for all requires? (no matter which we choose)